### ISSUE TITLE
Pub should notify that symlinks cannot be created on network shares

### ISSUE THREAD

**Issue by DartBot** [dart-lang/pub#415]
<a href="https://github.com/stevemessick"><img src="https://avatars.githubusercontent.com/u/8518285?v=3" align="left" width="96" height="96"hspace="10"></img></a> **Issue by [stevemessick](https://github.com/stevemessick)**
_Originally opened as dart-lang/sdk#8037_

----

Pub install does not work for project on network drive.

packages dosn't contain any directories
////////////////////////////////////////////////////////////////////////////////////
Editor: 0.2.10_r16761 (2013-01-07)
OS: Windows Vista - x86 (6.0)
JVM: 1.6.0_22
# projects: 1
# open dart files: 7
auto-run pub: true
mem max/total/free: 967 / 297 / 66 MB
thread count: 30
analysis: 0 tasks, 11 libraries, 1 contexts
index: 58567 relationships and 0 attributes in 12565 elements in 124 resources stored in 2 MB on disk

SDK installed: true
Dartium installed: true

**Comment by DartBot** [comment]
<a href="https://github.com/keertip"><img src="https://avatars.githubusercontent.com/u/2192312?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [keertip](https://github.com/keertip)**

----

*Set owner to @munificent.*
*Removed this from the [M4](../milestones/M4) milestone.*
*Removed [Area-Editor](../labels/Area-Editor) label.*
*Added [Area-Pub](../labels/Area-Pub) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

Unfortunately, this is a consequence of the fact that pub uses NTFS junction points to do symlinks on Windows. Those only work on local drives.

We won't be able to make this work, but we should notify the user of an error instead of silently failing. I'm repurposing this bug to that effect.
______
*Added this to the [Later](../milestones/Later) milestone.*
*Changed the title to: "Pub should notify that symlinks cannot be created on network shares".*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

Issue dart-lang/sdk#9239 has been merged into this issue.
______
 cc @sgjesse.

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Marked this as being blocked by dart-lang/sdk#15103.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

*Added [OpSys-Windows](../labels/OpSys-Windows) label.*

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

Issue dart-lang/sdk#17178 has been merged into this issue.
______
 cc @keertip.

**Comment by DartBot** [comment]
*This comment was originally written by mor...&#064;gmail.com*
_____
see my suggestion in the initial issue


2014-04-02 19:35 GMT+02:00 &lt;dart@googlecode.com>:

**Comment by DartBot** [comment]
<a href="https://github.com/nex3"><img src="https://avatars.githubusercontent.com/u/188?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [nex3](https://github.com/nex3)**

----

Issue dart-lang/sdk#17178 has been merged into this issue.

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

Issue dart-lang/sdk#17178 has been merged into this issue.

**Comment by DartBot** [comment]
<a href="https://github.com/munificent"><img src="https://avatars.githubusercontent.com/u/46275?v=3" align="left" width="48" height="48"hspace="10"></img></a> **Comment by [munificent](https://github.com/munificent)**

----

We're planning to move away from using symlinks at all, so we don't intend to put time into handle the lack of them more gracefully.
______
*Added [NotPlanned](../labels/NotPlanned) label.*
